draft-gss-keyex-sha2.xml

```diff
@@ -86,8 +86,12 @@
     <section title="Rationale">
       Due to security concerns with SHA-1 <xref target="RFC6194"/> and with
       MODP groups with less than 2048 bits <xref target="NIST-SP-800-131Ar1"/>
-      we propose the use of the SHA-2 based hashes with DH group14, group16
-      and group18 <xref target="RFC3526"/>. Following the rationale of <xref
+      we propose the use of the SHA-2 based hashes with DH group14, group15,
+      group16, group17
+      and group18 <xref target="RFC3526"/>. Additionally we add support for
```

Comment:
it was making for a cleaner diff until I did squash it together with the change that added the ecc groups